### PR TITLE
OH2-185 update patient return 500 error

### PR DESCRIPTION
**Pull request by ArnaudFonzam**
See OH2-185.

### CODE REVIEW

File: src/test/java/org/isf/opd/rest/OpdControllerTest.java [hunk 1]
```diff
@@ -46,6 +46,7 @@
 import org.mockito.MockitoAnnotations;
 import org.modelmapper.ModelMapper;
 import org.slf4j.Logger;
+import org.springframework.beans.factory.annotation.Autowired;
```
Comment (dbmalkovsky) [inline src/test/java/org/isf/opd/rest/OpdControllerTest.java:49]: Why was this import added?  `Autowired` doesn't exist in the file.

**Review by mwithi — APPROVED** [review]
LGTM. Thanks a lot!
